package.json

```diff
@@ -16,6 +16,7 @@
     "@react-navigation/stack": "^5.3.3",
     "base-64": "^0.1.0",
     "expo": "^37.0.0",
+    "axios": "^0.19.2",
```

Comment:
No need to use axios, we're pulling all data from Firebase.

Reply:
Okay, I was using the Public API to get a stat data. But if you have already on Firebase that's much better and efficient.